query/filter/filter.go

```diff
@@ -51,7 +51,7 @@ type Filter interface {
 	Matches(doc []byte) bool
 	// MatchesDoc similar to Matches but used when document is already parsed
 	MatchesDoc(doc map[string]interface{}) bool
-	ToSearchFilter() []string
```

Comment:
We are not sending multiple filters to search backend anymore. A single string filter for all and/or queries.